gef-ui/server/services/api.js

```diff
@@ -341,6 +341,31 @@ const downloadFile = async ({ fileId, userToken }) => {
   }
 };
 
+/**
+ * @param {Object} param
+ * @param {string} param.facilityId
+ * @param {string} param.amendmentId
+ * @param {string} param.userToken
+ * @returns {Promise<(import('@ukef/dtfs2-common').FacilityAmendmentWithUkefId)>}>}
+ */
+const getAmendment = async ({ facilityId, amendmentId, userToken }) => {
+  if (!isValidMongoId(facilityId)) {
+    throw new Error('Invalid facility ID');
+  }
+
+  if (!isValidMongoId(amendmentId)) {
+    throw new Error('Invalid amendment ID');
+  }
+
+  try {
+    const { data } = await Axios.get(`/gef/facilities/${facilityId}/amendments/${amendmentId}`, config(userToken));
+    return data;
+  } catch (error) {
+    console.error('Failed to get the amendment', error);
```

Comment:
all console errors with an object being printed should have %o at the end.  `'Failed to get the amendment %o'`

Reply:
Thank you - didn't quite understand how this worked before but I do now after some looking around. I've fixed this here.